### PR TITLE
Adapt predict_variance_derivatives to SMT codestyle

### PR DESCRIPTION
**Pull request by LDAP**

### CODE REVIEW

**Review by Paul-Saves — APPROVED** [review]
Seems great. My thanks to you.

File: smt/surrogate_models/surrogate_model.py [hunk 1]
```diff
@@ -384,15 +385,17 @@ def predict_variances(self, x):
...
+
         Returns:
         --------
-        - derived_variance: array_like
-        The jacobian of the variance of the kriging model
+        derived_variance: np.ndarray
+            The jacobian of the variance
         """
         check_support(self, "derivatives")
         check_support(self, "variances")
```
Comment (relf) [inline smt/surrogate_models/surrogate_model.py:401]: Much better thanks! Then you can replace the two lines above by <code>check_support(self, "variance_derivatives")</code>

**Review by relf — CHANGES_REQUESTED** [review]

File: smt/surrogate_models/surrogate_model.py (hunk 1 above)
Comment (LDAP) [reply smt/surrogate_models/surrogate_model.py:401]: Should be enough to delete them since check_support(...) is called in _predict_variance_derivatives?

Comment (LDAP) [reply smt/surrogate_models/surrogate_model.py:401]: I am going to add it to match the other methods.

**Review by relf — APPROVED** [review]

**Pull request merged**